src/styles/blocks/card.scss

```diff
@@ -0,0 +1,93 @@
+.card {
+  box-sizing: border-box;
+  border: 1px solid #e5e5e5;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  &__title {
+    color: $text-main-color;
+    margin: 0 0 4px;
+    font-weight: 500;
+    line-height: 18px;
+    font-size: 12px;
+  }
+
+  &__img {
+    display: block;
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__code {
+    color: $text-second-color;
+    font-weight: 400;
+    margin-bottom: 16px;
+  }
+
+  &__code,
+  &__review {
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__block {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin-bottom: 24px;
+
+    &-value {
+      color: $text-main-color;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+
+    &-text {
+      color: $text-second-color;
+      font-size: 12px;
+      line-height: 18px;
+    }
+
+    &-sum {
```

Comment:
nope, don't do it like that, it confuses devs, cause they might think that price is a block and not the element